services/galley/src/Galley/API/Error.hs

```diff
@@ -154,7 +154,8 @@ legalHoldServiceBadResponse :: Wai.Error
 legalHoldServiceBadResponse = Wai.mkError status400 "legalhold-status-bad" "legal hold service: invalid response"
 
 legalHoldNotEnabled :: Wai.Error
-legalHoldNotEnabled = Wai.mkError status403 "legalhold-not-enabled" "legal hold is not enabled for this team"
+legalHoldNotEnabled =
+  Wai.mkError status404 "legalhold-not-enabled" "legal hold is not enabled for this team"
```

Comment:
I don't think we should be changing this status code, because this error is used somewhere else as well.

Reply:
I checked and the Galley version wasn't used anywhere else, though Brig has an identical definition that is used.

Reply:
But, with the switch to MultiVerb, we put the 404 info elsewhere, so this change disappears.